### DISCUSSION TITLE
Transfer Education to different account?

### DISCUSSION THREAD

**Discussion by slweeb** [community/community#50867] [GitHub Education]
Hello!

Is it a possibility to transfer my GitHub Education enrollment from one GitHub account to another? I have access to both. ☺️

Is there an easy way? Or can I unenroll then re-enroll? Thank you :)

**Comment by samus-aran** [comment]
Hi @slweeb  :wave:!

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

Hope that helps 👍

**Reply by slweeb** [reply]
Hello!!

This is a bit disappointing because I want this to be my main account. Is it an option to remove myself from the program and reapply on this account?
